src/epd7in5b_v2/command.rs

```diff
@@ -0,0 +1,156 @@
+//! SPI Commands for the Waveshare 7.5"(B) V2 E-Ink Display
```

Comment:
```suggestion
//! SPI Commands for the Waveshare 7.5"(B) V2 and V3 E-Ink Display
```

Maybe include the v3 as well then if its already tested and working with the same commands.